### PR TITLE
[zh-cn] sync authorization.md

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/access-authn-authz/authorization.md

### CODE REVIEW

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 1]
```diff
@@ -10,72 +12,85 @@ reviewers:
...
+request can be allowed; however, Kubernetes also allows anonymous requests in
+some circumstances.
+
+For an overview of how authorization fits into the wider context of API access
+control, read
+[Controlling Access to the Kubernetes API](/docs/concepts/security/controlling-access/).
 -->
-了解有关 Kubernetes 鉴权的更多信息，包括使用支持的鉴权模块创建策略的详细信息。
+Kubernetes 鉴权在[身份认证](/zh-cn/docs/reference/access-authn-authz/authentication/)之后进行。
+通常，发出请求的客户端必须经过身份验证（登录）才能允许其请求；
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:36]:
```suggestion
通常，发出请求的客户端必须经过身份认证（登录），其请求才可以被允许；
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 2]
```diff
@@ -10,72 +12,85 @@ reviewers:
...
+但是，Kubernetes 在某些情况下也允许匿名请求。
 
+有关鉴权如何融入 API 访问控制的更多的上下文信息，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:39]:
```suggestion
有关鉴权在 API 访问控制中的位置这类进一步的语境信息，
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 3]
```diff
@@ -10,72 +12,85 @@ reviewers:
...
-When multiple authorization modules are configured, each is checked in sequence.
-If any authorizer approves or denies a request, that decision is immediately
-returned and no other authorizer is consulted. If all modules have no opinion on
-the request, then the request is denied. A deny returns an HTTP status code 403.
+Kubernetes admission control happens after authorization has completed (and,
+therefore, only when the authorization decision was to allow the request).
 -->
-（尽管 Kubernetes 使用 API 服务器，但是依赖于特定对象种类的特定字段的访问控制
-和策略由准入控制器处理。）
+依赖于特定对象种类的特定字段的{{< glossary_tooltip text="访问控制" term_id="admission-controller" >}}和策略由准入控制器处理。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:73]:
```suggestion
依赖于特定对象种类的特定字段的访问控制和策略由{{< glossary_tooltip text="准入控制器" term_id="admission-controller" >}}处理。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 4]
```diff
@@ -10,72 +12,85 @@ reviewers:
...
+
+Kubernetes 准入控制发生在鉴权完成后（因此，仅当鉴权决策是允许请求时）。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:75]:
```suggestion
Kubernetes 准入控制发生在鉴权完成之后（因此，仅当鉴权决策是允许请求时）。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 5]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+are considered _non-resource requests_, and use the lower-cased HTTP method of the request as the verb.
 
-For example, a `GET` request to endpoints like `/api` or `/healthz` would use `get` as the verb.
+For example, making a `GET` request to endpoints like `/api` or `/healthz` would use **get** as the verb.
 -->
-## 确定请求动词  {#determine-the-request-verb}
+### 请求动词和鉴权  {#determine-the-request-verb}
 
-**非资源请求**
+#### 非资源请求
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:141]:
```suggestion
#### 非资源请求   {#request-verb-non-resource}
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 6]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+#### Resource requests
 
-To determine the request verb for a resource API endpoint, review the HTTP verb
-used and whether or not the request acts on an individual resource or a
+To determine the request verb for a resource API endpoint, Kubernetes maps the HTTP verb
+used and considers whether or not the request acts on an individual resource or a
 collection of resources:
 -->
-**资源请求**
+#### 资源请求
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:156]:
```suggestion
#### 资源请求    {#resource-requests}
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 7]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
 HTTP 动词 | 请求动词
-----------|---------------
-POST      | create
-GET, HEAD | get （针对单个资源）、list（针对集合）
-PUT       | update
-PATCH     | patch
-DELETE    | delete（针对单个资源）、deletecollection（针对集合）
+--------------|---------------
+`POST`        | **create**
+`GET`, `HEAD` | **get**（for individual resources）, **list**（针对集合，包括完整的对象内容）, **watch**（用于查看单个资源或资源集合）
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:173]:
```suggestion
`GET`, `HEAD` | **get**（针对单个资源）、**list**（针对集合，包括完整的对象内容）、**watch**（用于查看单个资源或资源集合）
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 8]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+`PUT`         | **update**
+`PATCH`       | **patch**
+`DELETE`      | **delete**（针对单个资源）, **deletecollection**（针对集合）
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:176]:
```suggestion
`DELETE`      | **delete**（针对单个资源）、**deletecollection**（针对集合）
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 9]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+
+<!--
+## Authorization context
+
+Kubernetes expects attributes that are common to REST API requests. This means
+that Kubernetes authorization works with existing organization-wide or
+cloud-provider-wide access control systems which may handle other APIs besides
+the Kubernetes API.
+-->
+## 鉴权上下文
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:215]:
```suggestion
## 鉴权上下文    {#authorization-context}
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 10]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
-   * When specified RBAC (Role-Based Access Control) uses the `rbac.authorization.k8s.io` API group to drive authorization decisions, allowing admins to dynamically configure permission policies through the Kubernetes API.
-   * To enable RBAC, start the apiserver with `--authorization-mode=RBAC`.
- * **Webhook** - A WebHook is an HTTP callback: an HTTP POST that occurs when something happens; a simple event-notification via HTTP POST. A web application implementing WebHooks will POST a message to a URL when certain things happen. To learn more about using the Webhook mode, see [Webhook Mode](/docs/reference/access-authn-authz/webhook/).
+`ABAC` ([attribute-based access control](/docs/reference/access-authn-authz/abac/))
+: Kubernetes ABAC mode defines an access control paradigm whereby access rights are granted to users through the use of policies which combine attributes together. The policies can use any type of attributes (user attributes, resource attributes, object, environment attributes, etc).
 -->
 ## 鉴权模块  {#authorization-modules}
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:233]:
```suggestion
## 鉴权模式  {#authorization-modules}
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 11]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
-* **Webhook** —— WebHook 是一个 HTTP 回调：发生某些事情时调用的 HTTP POST；
-  通过 HTTP POST 进行简单的事件通知。
-  实现 WebHook 的 Web 应用程序会在发生某些事情时将消息发布到 URL。
-  要了解有关使用 Webhook 模式的更多信息，请参阅
-  [Webhook 模式](/zh-cn/docs/reference/access-authn-authz/webhook/)。
-
-<!--
-#### Checking API Access
+`AlwaysAllow`
+: 此模式允许所有请求，但存 [安全风险](#warning-always-allow)，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:236]:
```suggestion
: 此模式允许所有请求，但存在[安全风险](#warning-always-allow)，
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 12]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+  仅当你的 API 请求不需要鉴权时（例如，用于测试），才使用此鉴权模式。
+
+`AlwaysDeny`
+: 此模式阻止所有请求。仅将此鉴权模式用于测试。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:240]:
```suggestion
: 此模式阻止所有请求。此鉴权模式仅适用于测试。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 13]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+  In this mode, Kubernetes uses the `rbac.authorization.k8s.io` API group to drive authorization decisions, allowing you to dynamically configure permission policies through the Kubernetes API.
+
+`Node`
+: A special-purpose authorization mode that grants permissions to kubelets based on the pods they are scheduled to run. To learn more about the Node authorization mode, see [Node Authorization](/docs/reference/access-authn-authz/node/).
+
+`Webhook`
+: Kubernetes [webhook mode](/docs/reference/access-authn-authz/webhook/) for authorization makes a synchronous HTTP callout, blocking the request until the remote HTTP service responds to the query.You can write your own software to handle the callout, or use solutions from the ecosystem.
+-->
+`RBAC`（[基于角色的访问控制](/zh-cn/docs/reference/access-authn-authz/rbac/)）
+: Kubernetes RBAC 是一种根据企业内各个用户的角色来管理对计算机或网络资源的访问权限的方法。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:258]:
```suggestion
: Kubernetes RBAC 是一种根据企业内各个用户的角色来管理其对计算机或网络资源的访问权限的方法。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 14]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+  在这种模式下，Kubernetes 使用 `rbac.authorization.k8s.io` API 组来驱动鉴权决策，
+  允许你通过 Kubernetes API 动态配置权限策略。
+
+`Node`
+: 一种特殊用途的鉴权模式，根据 kubelet 计划运行的 Pod 向其授予权限。
+  要了解有关 Node 鉴权模式的更多信息，请参阅 [Node 鉴权](/zh-cn/docs/reference/access-authn-authz/node/)。
+
+`Webhook`
+: Kubernetes 的 [Webhook 鉴权模式](/docs/reference/access-authn-authz/webhook/)用于鉴权，进行同步 HTTP 调用，
+  阻止请求直到远程 HTTP 服务响应查询。你可以编写自己的软件来处理调出，也可以使用生态系统中的解决方案。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:269]:
```suggestion
  阻塞请求直到远程 HTTP 服务响应查询。你可以编写自己的软件来处理这种向外调用，也可以使用生态系统中的解决方案。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 15]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+
+You should not use the `AlwaysAllow` mode on a Kubernetes cluster where the API server
+is reachable from the public internet.
+-->
+启用 `AlwaysAllow` 模式会绕过鉴权；请勿在你不信任**所有**潜在 API
+客户端（包括你运行的工作负载）的集群上使用该模式。
+
+鉴权机制通常返回“拒绝”或“无意见”的结果；
+有关更多信息，请参阅[鉴权裁决](#determine-whether-a-request-is-allowed-or-denied)。
+激活 `AlwaysAllow` 意味着如果所有其他鉴权者都返回 “no opinion”，则允许该请求。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:293]:
```suggestion
激活 `AlwaysAllow` 意味着如果所有其他鉴权者都返回“无意见”，则允许该请求。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 16]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+
+You have to pick one of the two configuration approaches; setting both `--authorization-config`
+path and configuring an authorization webhook using the `--authorization-mode` and
+`--authorization-webhook-*` command line arguments is not allowed.
+If you try this, the API server reports an error message during startup, then exits immediately.
+-->
+### 鉴权模式配置 {#choice-of-authz-config}
+
+你可以仅使用[命令行参数](#using-flags-for-your-authorization-module)或作为 Beta
+特性使用[配置文件](#using-configuration-file-for-authorization)来配置 Kubernetes API 服务器的鉴权链。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:315]:
```suggestion
你可以仅使用[命令行参数](#using-flags-for-your-authorization-module)，
或使用[配置文件](#using-configuration-file-for-authorization)来配置 Kubernetes API
服务器的鉴权链，后者目前是 Beta 特性。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 17]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+* `--authorization-mode=AlwaysDeny` (always denies requests)
+
+You can choose more than one authorization mode; for example:
+`--authorization-mode=Node,Webhook`
+-->
+你可以使用以下模式：
+
+* `--authorization-mode=ABAC`（基于属性的访问控制模式）
+* `--authorization-mode=RBAC`（基于角色的访问控制模式）
+* `--authorization-mode=Node`（节点鉴权者）
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:345]:
```suggestion
* `--authorization-mode=Node`（节点鉴权组件）
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 18]
```diff
@@ -112,111 +127,600 @@ Kubernetes 仅审查以下 API 请求属性：
...
+webhooks. The authorization items in that chain can have well-defined parameters that validate
+requests in a particular order, offering you fine-grained control, such as explicit Deny on failures.
+
+The configuration file approach even allows you to specify
+[CEL](/docs/reference/using-api/cel/) rules to pre-filter requests before they are dispatched
+to webhooks, helping you to prevent unnecessary invocations. The API server also automatically
+reloads the authorizer chain when the configuration file is modified.
+-->
+作为一项 Beta 级别特性，Kubernetes 允许你配置可包含多个 Webhook 的鉴权链。
+该链中的鉴权项可以具有明确定义的参数，这些参数可以按特定顺序验证请求，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:395]:
```suggestion
该链中的鉴权项可以具有明确定义的参数，这些参数可以按特定顺序检查请求，
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 19]
```diff
@@ -112,111 +127,601 @@ Kubernetes 仅审查以下 API 请求属性：
...
+如果你想使用命令行参数而不是配置文件，这也是一种有效且受支持的方法。
+某些鉴权功能（例如：多个 Webhook、Webhook 失败策略和预过滤规则）仅在使用鉴权配置文件时可用。
+
+<!--
+#### Example configuration {#authz-config-example}
+-->
+#### 示例配置   {#authz-config-example}
+
+{{< highlight yaml "linenos=false,hl_lines=2-4" >}}
+<!--
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:422]: image

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 20]
```diff
@@ -112,111 +127,601 @@ Kubernetes 仅审查以下 API 请求属性：
...
+    name: in-cluster-authorizer
+    webhook:
+      authorizedTTL: 5m
+      unauthorizedTTL: 30s
+      timeout: 3s
+      subjectAccessReviewVersion: v1
+      failurePolicy: NoOpinion
+      connectionInfo:
+        type: InClusterConfig
+-->
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:522]: image

**Review by mengjiao-liu — CHANGES_REQUESTED** [review]
Please check the format from the [preview page](https://deploy-preview-46537--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/access-authn-authz/authorization/).

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 21]
```diff
@@ -112,111 +127,601 @@ Kubernetes 仅审查以下 API 请求属性：
...
+
+You should not use the `AlwaysAllow` mode on a Kubernetes cluster where the API server
+is reachable from the public internet.
+-->
+启用 `AlwaysAllow` 模式会绕过鉴权；请勿在你不信任**所有**潜在 API
+客户端（包括你运行的工作负载）的集群上使用该模式。
+
+鉴权机制通常返回“拒绝”或“无意见”的结果；
+有关更多信息，请参阅[鉴权裁决](#determine-whether-a-request-is-allowed-or-denied)。
+激活 `AlwaysAllow` 意味着如果所有其他鉴权者都返回“无意见”，则允许该请求。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:293]:
```suggestion
激活 `AlwaysAllow` 意味着如果所有其他鉴权器都返回“无意见”，则允许该请求。
```
“ authorizer” -> "鉴权器"
根据中文属于对照：https://kubernetes.io/zh-cn/docs/contribute/localization_zh/#glossary

下同。

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 22]
```diff
@@ -112,111 +127,601 @@ Kubernetes 仅审查以下 API 请求属性：
...
+authorizers:
+  - type: Webhook
+    # 用于描述鉴权人的名称
+    # 这明确用于监控机制的指标
+    # 注意
+    #   - 该字段的验证与今天的 K8s 标签的验证方式类似。
+    # 必填，无默认值
+    name: webhook
+    webhook:
+      # 缓存来自 Webhook 鉴权者的“鉴权”响应的持续时间
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:538]:
```suggestion
      # 缓存来自 Webhook 鉴权器的“鉴权”响应的持续时间
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 23]
```diff
@@ -112,111 +127,601 @@ Kubernetes 仅审查以下 API 请求属性：
...
+      # 与设置 `--authorization-webhook-cache-authorized-ttl` 标志相同
+      # 默认值：5m0s
+      authorizedTTL: 30s
+      # 缓存来自 Webhook 鉴权者的“未授权”响应的持续时间。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:542]:
```suggestion
      # 缓存来自 Webhook 鉴权器的“未授权”响应的持续时间。
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 24]
```diff
@@ -112,111 +127,601 @@ Kubernetes 仅审查以下 API 请求属性：
...
+      # 必填，无默认值
+      # 有效值：v1beta1、v1
+      subjectAccessReviewVersion: v1
+      # MatchConditionSubjectAccessReviewVersion 指定评估 CEL 表达式的 SubjectAccessReview 版本
+      # 有效值：v1
+      # 必填，无默认值
+      matchConditionSubjectAccessReviewVersion: v1
+      # 当 Webhook 请求无法完成或返回格式错误的响应或评估 matchConditions 时出现错误时，控制鉴权决定。
+      # 有效值：
+      #   - NoOpinion：继续联系后续鉴权者，看其中是否有人允许该请求
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:561]:
```suggestion
      #   - NoOpinion：继续联系后续鉴权器，看其中是否有人允许该请求
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 25]
```diff
@@ -112,111 +127,601 @@ Kubernetes 仅审查以下 API 请求属性：
...
+      #   - Deny：拒绝请求而不咨询后续鉴权者
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:562]:
```suggestion
      #   - Deny：拒绝请求而不咨询后续鉴权器
```

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 26]
```diff
@@ -112,111 +127,601 @@ Kubernetes 仅审查以下 API 请求属性：
...
+{{< note >}}
+<!--
+You must ensure that all non-webhook authorizer types remain unchanged in the file on reload.
+
+A reload **must not** add or remove Node or RBAC authorizers (they can be reordered,
+but cannot be added or removed).
+-->
+你必须确保重新加载时文件中所有非 Webhook 鉴权器类型保持不变。
+
+重新加载**不能**添加或删除节点或 RBAC 鉴权者（可以重新排序，但不能添加或删除）。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:649]:
```suggestion
重新加载**不能**添加或删除节点或 RBAC 鉴权器（可以重新排序，但不能添加或删除）。
```

**Review by mengjiao-liu — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md (hunk 21 above)
Comment (tengqm) [reply content/zh-cn/docs/reference/access-authn-authz/authorization.md:293]: “鉴权器”的翻译有点儿生硬，是不是意译为“鉴权组件”更合适？

Comment (mengjiao-liu) [reply content/zh-cn/docs/reference/access-authn-authz/authorization.md]: 也可以。

File: content/zh-cn/docs/reference/access-authn-authz/authorization.md [hunk 27]
```diff
@@ -112,111 +127,601 @@ Kubernetes 仅审查以下 API 请求属性：
...
+如果没有观察到更改事件，则也会按照 60 秒的计划重新加载。
+
+{{< note >}}
+<!--
+You must ensure that all non-webhook authorizer types remain unchanged in the file on reload.
+
+A reload **must not** add or remove Node or RBAC authorizers (they can be reordered,
+but cannot be added or removed).
+-->
+你必须确保重新加载时文件中所有非 Webhook 鉴权器类型保持不变。
```
Comment (mengjiao-liu) [inline content/zh-cn/docs/reference/access-authn-authz/authorization.md:647]:
```suggestion
你必须确保重新加载时文件中所有非 Webhook 鉴权组件类型保持不变。
```
全文保持一致。

**Review by mengjiao-liu — COMMENTED** [review]
/approve

**Pull request merged**